hawkmoth/__init__.py

```diff
@@ -46,7 +46,10 @@ def __parse(self, viewlist, filename):
         compat = self.options.get('compat', env.config.cautodoc_compat)
         clang = self.options.get('clang', env.config.cautodoc_clang)
 
-        comments = parse(filename, compat=compat, clang=clang)
+        comments, errors = parse(filename, compat=compat, clang=clang)
+
+        for (ffile, line, msg) in errors:
+            env.app.warn(_err_fmt.format(msg, location=(ffile, line)))
```

Comment:
This doesn't work right now (not sure why actually, but didn't dwell on it either, this is an RFC).

@jnikula I actually don't know where this comes from, but I saw it a few lines below in `run` and thought it might be the correct way to pass warnings/ errors to Sphinx.